### PR TITLE
Content host test cleanup

### PR DESCRIPTION
**Pull request by shweta83**
Host-Content Cleanup:
1. Update some fixtures and remove some duplicate fixtures created in this module
2. Replace entities.

### CODE REVIEW

**Comment by shweta83** [comment]
> @shweta83 Hey Shweta! Even I was plannig to update test_client upgrade scenarios in our call today. BTW, whats you plan to achieve here?
> 
> My plan was this:
> 
> ```
> - Remove use of execute and docker_execute_command as VM would be checked out from broker container and we could use host methods to replace
> - entities to be replaced with target_sat
> - dockerize to be replaced with broker container content host
> ```

@jyejare I am working on following scope for this PR
1. Update some fixtures and remove some duplicate fixtures created in this module
2. Replace entities.

**Review by adarshdubey-star — APPROVED** [review]
Ack

File: tests/upgrades/test_client.py [hunk 1]
```diff
@@ -47,22 +45,7 @@
...
-    return (
-        entities.LifecycleEnvironment()
-        .search(query={'search': f'name={ENVIRONMENT} and organization_id={default_org.id}'})[0]
-        .read()
-    )
-
-
-@pytest.fixture(scope='module')
-def sat6tools_repo(default_org):
+def sat6tools_repo(default_org, target_sat):
```
Comment (Gauravtalreja1) [inline tests/upgrades/test_client.py:48]: There seems to be no use for this feature, can we remove it?

Comment (shweta83) [reply tests/upgrades/test_client.py]: Sure, we can. Updated.

File: tests/upgrades/test_client.py [hunk 2]
```diff
@@ -23,82 +23,45 @@
...
-        releasever=None,
-    )
-    sat6tools_repo = entities.Repository(id=sat6tools_repo_id).read()
-    assert sat6tools_repo.sync()['result'] == 'success'
-    return sat6tools_repo
-
-
-@pytest.fixture(scope='module')
-def pre_upgrade_repo(default_org, module_product):
+def pre_upgrade_repo(default_org, module_product, target_sat):
```
Comment (jyejare) [inline tests/upgrades/test_client.py:42]:
```suggestion
def pre_upgrade_repo(default_org, module_product, module_target_sat):
```

File: tests/upgrades/test_client.py [hunk 3]
```diff
@@ -23,82 +23,45 @@
...
+    post_upgrade_repo = module_target_sat.api.Repository(
         url=settings.repos.yum_9.url, product=module_product
     ).create()
     assert post_upgrade_repo.sync()['result'] == 'success'
     return post_upgrade_repo
 
 
 @pytest.fixture(scope='module')
-def pre_upgrade_cv(default_org, pre_upgrade_repo):
+def pre_upgrade_cv(default_org, pre_upgrade_repo, target_sat):
```
Comment (jyejare) [inline tests/upgrades/test_client.py:62]:
```suggestion
def pre_upgrade_cv(default_org, pre_upgrade_repo, module_target_sat):
```

File: tests/upgrades/test_client.py [hunk 4]
```diff
@@ -107,9 +70,9 @@ def pre_upgrade_cv(default_org, pre_upgrade_repo):
 
 
 @pytest.fixture(scope='module')
-def post_upgrade_cv(default_org, post_upgrade_repo):
+def post_upgrade_cv(default_org, post_upgrade_repo, target_sat):
```
Comment (jyejare) [inline tests/upgrades/test_client.py:73]:
```suggestion
def post_upgrade_cv(default_org, post_upgrade_repo, module_target_sat):
```

**Review by jyejare — CHANGES_REQUESTED** [review]
The scope of fixtures matters !

**Comment by shweta83** [comment]
> The scope of fixtures matters !

Updated the fixtures.

**Review by ColeHiggins2 — APPROVED** [review]
ACK

File: tests/upgrades/test_client.py [hunk 5]
```diff
@@ -23,82 +23,45 @@
...
         url=settings.repos.yum_0.url, product=module_product
     ).create()
     assert pre_upgrade_repo.sync()['result'] == 'success'
     return pre_upgrade_repo
 
 
 @pytest.fixture(scope='module')
-def post_upgrade_repo(default_org, module_product):
+def post_upgrade_repo(default_org, module_product, module_target_sat):
     """Enable custom errata repository"""
```
Comment (Gauravtalreja1) [inline tests/upgrades/test_client.py:53]: Same here, `default_org` is not in use. Can this be removed?

**Review by Gauravtalreja1 — COMMENTED** [review]
Ack, few comments

File: tests/upgrades/test_client.py (hunk 5 above)
Comment (shweta83) [reply tests/upgrades/test_client.py]: Updated.

File: tests/upgrades/test_client.py [hunk 6]
```diff
@@ -23,82 +23,45 @@
...
-        releasever=None,
-    )
-    sat6tools_repo = entities.Repository(id=sat6tools_repo_id).read()
-    assert sat6tools_repo.sync()['result'] == 'success'
-    return sat6tools_repo
-
-
-@pytest.fixture(scope='module')
-def pre_upgrade_repo(default_org, module_product):
+def pre_upgrade_repo(default_org, module_product, module_target_sat):
```
[the remark this answers is not in the document]
Comment (shweta83) [reply tests/upgrades/test_client.py]: Updated.

**Review by jyejare — APPROVED** [review]

**Pull request merged**